src/core/razorpay-base.ts

```diff
@@ -161,6 +161,12 @@ abstract class RazorpayBase extends AbstractPaymentProcessor {
       ...intentRequestData,
     };
 
+    if(!(customer && customer?.billing_address?.phone && customer?.phone ))
```

Comment:
```suggestion
    if(!(customer?.billing_address?.phone || customer?.phone ))
```
We are failing with new version, I think this should fix it

Reply:
hey thanks for fixing this. have you tested your fix ?

Reply:
thanks fixed it